pom.xml

```diff
@@ -119,8 +119,8 @@
     <version.org.apache.felix.scr.annotations>1.7.0</version.org.apache.felix.scr.annotations>
     <version.org.apache.geronimo.specs.geronimo-jta_1.1_spec>1.1.1</version.org.apache.geronimo.specs.geronimo-jta_1.1_spec>
     <version.org.apache.geronimo.specs.geronimo-servlet_3.0_spec>1.0</version.org.apache.geronimo.specs.geronimo-servlet_3.0_spec>
-    <version.org.apache.httpcomponents.httpclient>4.4-beta1</version.org.apache.httpcomponents.httpclient>
-    <version.org.apache.httpcomponents.httpcore>4.4-beta1</version.org.apache.httpcomponents.httpcore>
+    <version.org.apache.httpcomponents.httpclient>4.4.1</version.org.apache.httpcomponents.httpclient>
+    <version.org.apache.httpcomponents.httpcore>4.4.1</version.org.apache.httpcomponents.httpcore>
```

Comment:
All seems to be fine, yes.
On 21 May 2015 16:29, "Eric Wittmann" <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/apiman/apiman/pull/127#discussion_r30813942>:
>
> > @@ -119,8 +119,8 @@
> >      <version.org.apache.felix.scr.annotations>1.7.0</version.org.apache.felix.scr.annotations>
> >      <version.org.apache.geronimo.specs.geronimo-jta_1.1_spec>1.1.1</version.org.apache.geronimo.specs.geronimo-jta_1.1_spec>
> >      <version.org.apache.geronimo.specs.geronimo-servlet_3.0_spec>1.0</version.org.apache.geronimo.specs.geronimo-servlet_3.0_spec>
> > -    <version.org.apache.httpcomponents.httpclient>4.4-beta1</version.org.apache.httpcomponents.httpclient>
> > -    <version.org.apache.httpcomponents.httpcore>4.4-beta1</version.org.apache.httpcomponents.httpcore>
> > +    <version.org.apache.httpcomponents.httpclient>4.4.1</version.org.apache.httpcomponents.httpclient>
> > +    <version.org.apache.httpcomponents.httpcore>4.4.1</version.org.apache.httpcomponents.httpcore>
>
> Nice - everything else still works I assume? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiman/apiman/pull/127/files#r30813942>.
>